charts/vela-core/templates/defwithtemplate/cpuhpa.yaml

```diff
@@ -38,8 +38,14 @@ spec:
         	}
         }
         parameter: {
-        	min:     *1 | int
-        	max:     *10 | int
+        
+        	// +usage=specify the min number of replicas to which the autoscaler can scale down
```

Comment:
Look the generated docs or cli table, capitalizing the first word look better:)

Reply:
got it, align with other definition is necessary